apps/mobile/src/features/token/components/token-loading.tsx

```diff
@@ -0,0 +1,52 @@
+import { Screen } from '@/components/screen/screen';
```

Comment:
For SIP-10 - the `asset` needed to fetch other data relies on `balance` being loaded so it can be a bit jumpy. It's strange as we should already have it cached from the home screen.

I added this improvised loading screen for SIP-10's to make it more smooth, I will check it with design but it's an improvement for now on a blank screen and jumpy content:

<img width="1206" height="2622" alt="Simulator Screenshot - iPhone 16 iOS 18 4 - 2025-08-26 at 11 50 45" src="https://github.com/user-attachments/assets/0b670e81-d951-48bc-9344-d1147ae7aefe" />